package.json

```diff
@@ -51,6 +51,7 @@
     "remix-forms": "^0.16.1",
     "remix-utils": "^2.11.1",
     "ts-node": "^10.7.0",
+    "uuid": "^8.3.2",
```

Comment:
No need to use a package. Nodejs has `crypto.randomUUID()`

Reply:
Changed in ce95081